engine/include/Engine/Core/Tools/Format.hpp

```diff
@@ -6,8 +6,6 @@
 
 #pragma once
 
-#include <cassert>
-#include <memory>
```

Comment:
I would rather see all the headers together next to each others, rather than split